debian/libspdk3.install

```diff
@@ -1 +1,2 @@
-lib/x86_64-linux-gnu/lib*.so.*
\ No newline at end of file
+lib/x86_64-linux-gnu/lib*.so.*
+scripts/{setup,common}.sh usr/share/spdk/scripts
```

Comment:
These cannot be in this package.  `lib*<SOVERSION>` packages in Ubuntu (and SUSE FWIW) must only contain versioned shared libs and no other files.  This is to allow multiple shared library (i.e. `libspdk3` and say, `libspdk4`) packages to be installed at the same time.  That works because there are no common files between those two packages to conflict.

Adding these scripts to this package violates that requirement and would make trying to install a `libspdk4` along with a `libspdk3` fail due to conflicts on the `/usr/share/spdk/scripts/` directory and files.

I would think these should go in the `spdk-tools` package, no?

Reply:
Done